doc/examples/applications/plot_3d_image_processing.py

```diff
@@ -0,0 +1,294 @@
+
+This tutorial is an introduction to three-dimensional image processing. Images
+are represented as `numpy` arrays. A single-channel, or grayscale, image is a
+2D matrix of pixel intensities of shape `(row, column)`. We can construct a 3D
+volume as a series of 2D planes, giving 3D images the shape
+`(plane, row, column)`. A multichannel, or RGB(A), image has an additional
+`channel` dimension in the final position containing color information.
+
+These conventions are summarized in the table below:
+
+=============== ===============================
+Image type      Coordinates
+=============== ===============================
+2D grayscale    `(row, column)`
+2D multichannel `(row, column, channel)`
+3D grayscale    `(plane, row, column)`
+3D multichannel `(plane, row, column, channel)`
+=============== ===============================
+
+Some 3D images are constructed with equal resolution in each dimension (e.g.,
+computer-generated rendering of a sphere). Most experimental data are captured
+with a lower resolution in one of the three dimensions, e.g., photographing
+thin slices to approximate a 3D structure as a stack of 2D images.
+The distance between pixels in each dimension, called spacing, is encoded as a
+tuple and is accepted as a parameter by some `skimage` functions and can be
+used to adjust contributions to filters.
+
+The data used in this tutorial were provided by the Allen Institute for Cell
+Science. They were downsampled by a factor of 4 in the `row` and `column`
+dimensions to reduce their size and, hence, computational time. The spacing
+information was reported by the microscope used to image the cells.
+
+"""
+
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d.art3d import Poly3DCollection
+import numpy as np
+from scipy import ndimage as ndi
+
+from skimage import (
+    exposure, io
+)
+from skimage.data import image_fetcher
+
+
+#####################################################################
+# Load and display 3D images
+# ==========================
+# Three-dimensional data can be loaded with `io.imread`.
+
+path = image_fetcher.fetch('data/cells.tif')
+data = io.imread(path)
+
+print("shape: {}".format(data.shape))
+print("dtype: {}".format(data.dtype))
+print("range: ({}, {})".format(data.min(), data.max()))
+
+# Report spacing from microscope
+original_spacing = np.array([0.2900000, 0.0650000, 0.0650000])
+
+# Account for downsampling of slices by 4
+rescaled_spacing = original_spacing * [1, 4, 4]
+
+# Normalize spacing so that pixels are a distance of 1 apart
+spacing = rescaled_spacing / rescaled_spacing[2]
+
+print("microscope spacing: {}\n".format(original_spacing))
+print("rescaled spacing: {} (after downsampling)\n".format(rescaled_spacing))
+print("normalized spacing: {}\n".format(spacing))
+
+#####################################################################
+# Let us try and visualize the (3D) image with `io.imshow`.
+
+try:
+    io.imshow(data, cmap="gray")
```

Comment:
this empty figure is used for the gallery thumbnail, please select another figure as in https://github.com/matplotlib/matplotlib/issues/17970

Reply:
Thanks, when building the docs I've only checked the page I was working on. I didn't think of checking the resulting gallery page. I'll add this small step to my mental workflow.

Reply:
> I'll add this small step to my mental workflow.

I thought I would add it to the contributor guide: #4857

Reply:
Done b5e6e4d327f. I hesitated quite a bit, because the 3rd figure is much more representative of 'exploring 3D images' but the 30 slices are way too tiny on a thumbnail... I thus chose the 4th figure, which is about adjusting exposure but it *looks* nice on a thumbnail. Plus it shows cells pretty clearly, which may catch the eye of a biologist or two :-)